src/command_line_parser.cc

```diff
@@ -1016,7 +1022,12 @@ CLParser::ParseCommandLine(int argc, char** argv)
           break;
         }
         case long_option_idx_base + 7: {
-          params_->using_concurrency_range = true;
+          if (params_->inference_load_mode != InferenceLoadMode::None) {
+            Usage(
+                "Cannot use both " + to_string(params_->inference_load_mode) +
+                " and Concurrency inference load modes.");
+          }
+          params_->inference_load_mode = InferenceLoadMode::Concurrency;
```

Comment:
Inference load modes now check at the time they are detected whether more than one was used, and errors out if so.